docs/docsite/rst/porting_guides/porting_guide_2.8.rst

```diff
@@ -232,6 +245,49 @@ Plugins
 
 * Play recap now counts ``ignored`` and ``rescued`` tasks as well as ``ok``, ``changed``, ``unreachable``, ``failed`` and ``skipped`` tasks, thanks to two additional stat counters in the ``default`` callback plugin. Tasks that fail and have ``ignore_errors: yes`` set are listed as ``ignored``. Tasks that fail and then execute a rescue section are listed as ``rescued``. Note that ``rescued`` tasks are no longer counted as ``failed`` as in Ansible 2.7 (and earlier).
 
+* Cache plugins can now be used for caching inventory. To make a new or custom cache plugin compatible with inventory
```

Comment:
This section seems awfully long for the porting guide. I'd move anything that isn't already covered in the `developing_inventory` page to there, and link to it from here. That way folks will be able to find this info easily even when 2.8 starts to fade in the rear-view mirror.

Reply:
Good point!